fuel-vm/tests/backtrace.rs

```diff
@@ -50,7 +50,7 @@ fn backtrace() {
 
     #[rustfmt::skip]
     let mut function_call = vec![
-        op::movi(0x10,  (contract_undefined.as_ref().len() + WORD_SIZE * 2) as Immediate18),
+        op::movi(0x10,  (contract_undefined.as_ref().len() + WORD_SIZE * 2 + 1) as Immediate18),
```

Comment:
Are these changes the result of those byte offsets you mentioned in our chat?

Reply:
It's because of the hp not being set correctly.